uint16 id
uint32[] src_img
uint32[] detect_img
uint16 detect_res
string detect_res_string
string create_time
